C/CasADi/build_tarballs.jl

```diff
@@ -0,0 +1,81 @@
+using BinaryBuilder, Pkg
+
+name = "CasADi"
+
+version = v"3.5.5"
+
+sources = [
+    GitSource(
+        "https://github.com/casadi/casadi.git",
+        "fadc86444f3c7ab824dc3f2d91d4c0cfe7f9dad5",
+    ),
+    DirectorySource("./bundled"),
+]
+
+script = raw"""
+cd $WORKSPACE/srcdir/casadi
+install_license LICENSE.txt
+
+mkdir -p build
+cd build
+
+export CXXFLAGS="${CXXFLAGS} -std=c++11"
```

Comment:
I think that maybe you were missing to make the armv7l build successful
```suggestion
export CXXFLAGS="-fPIC -std=c++11"
```
I didn't see what was the problem with the other linux platforms.

Side note: it's the package that should set the C++ standard with [`CXX_STANDARD`](https://cmake.org/cmake/help/latest/prop_tgt/CXX_STANDARD.html#prop_tgt:CXX_STANDARD), packagers should never do that manually.  If they have, it's upstream's fault.

Reply:
They nominally have a check for the standard, https://github.com/casadi/casadi/blob/402fe583f0d3cf1fc77d1e1ac933f75d86083124/CMakeLists.txt#L107-L164, but I hit some errors that only resolved when I explicitly set the standard.